features/polyExplorer/src/components/clusterStory/clusterStory.css

```diff
@@ -0,0 +1,67 @@
+.cluster-story .content {
+    width: 100%;
+    padding: 0 16px;
+    box-sizing: border-box;
+    z-index: 2;
+}
+
+.cluster-story .content * {
+    max-width: var(--max-width);
+    color: var(--color-text-dark);
+}
+
+.cluster-story .content p {
+    line-height: 120%;
+    letter-spacing: -0.01em;
+}
+
+.cluster-story .scroll-up-btn,
+.cluster-story .progress-bar {
+    position: absolute;
+    top: 0;
+    left: 0;
+}
+
+.cluster-story .progress-bar {
+    height: 4px;
+}
+
+.cluster-story .scroll-up-btn {
+    width: 100%;
+    height: 10%;
+}
+
+.cluster-story .content::-webkit-scrollbar-track {
+    margin-top: 8px;
+}
+
+.cluster-story-main-title {
+    font-size: 24px;
+    font-weight: 600;
+    line-height: 120%;
+    text-align: center;
+    letter-spacing: -0.01em;
+    margin: 0 0 25px 0;
+    padding-top: 25px;
+}
+
+.cluster-story-img {
+    width: 348px;
+    height: 468px;
+    display: block;
+    margin: 0 auto 60px auto;
+}
+
+.big-first-letter::first-letter {
+    float: left;
+    font-size: 60px;
+    line-height: 40px;
+    padding: 0 12px;
+}
+
+.cluster-story-title {
+    margin-top: 50px;
+    font-weight: bold;
+    font-size: 45px;
```

Comment:
Shouldn't we start to always use variables for font sizes and weights? Not only here, possibly everywhere...

Reply:
We could do that, but I think it's better if we do it after Lena's font unification because some of this `font-weight` will disappear or will be replaced by other numbers.